fe/src/components/Statistics/Statistics.re

```diff
@@ -0,0 +1,6 @@
+let component = ReasonReact.statelessComponent("Statistics");
```

Comment:
Ten komponent jest trochę bez sensu bo tylko wywołuje <StatisticsChartContainer

Reply:
Założyłem w tym przypadku, że do tego place będzie można dodać inne statystyki. W końcu sam wykres punktów zbiorowych nie musi stanowić całości tego działu :). A place to nawet chyba powinien być jako osobny komponent. Czyli rozdzielenie place od poszczególnych funkcjonalności.